rct_optimizations/include/rct_optimizations/noise_qualifier.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+#include <Eigen/Dense>
+#include "rct_optimizations/types.h"
+
+namespace rct_optimizations
+{
+
+/**
+  * @brief The NoiseStatistics struct output structure for image analysis data
+  */
+ struct NoiseStatistics
+ {
+   Eigen::VectorXd mean;
+   Eigen::VectorXd std_dev;
```

Comment:
I'm torn about how to represent mean/std. dev. pairs. I don't like maintaining two vectors that are assumed to be the same length. I also don't really like the approach I took in #51 where I used a `std::vector<std::pair<double, double>>` to represent the pair. Maybe we make a simple new struct with mean and std. dev. variables and then keep vectors of that struct. Any thoughts?

Reply:
Each Vector currently has 6 elements for x,y,z roll, pitch, yaw. This could be stored in 2 pose6d: one for mean, and one for standard deviation

Reply:
Eventually I would like to remove `Pose6d` and use Eigen objects instead, so I wouldn't suggest going that route

Reply:
Understood. In that case, I think a simple struct comprised of a mean and std dev is probably the best option. Should that be defined in noise_qualifier, or should I put it in types.h for future applications?

Reply:
For now, just put it inside the noise qualification header. We can amend #51 later to use the same class